release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -78,6 +78,14 @@ For more information, see xref:../installing/installing_ibm_z/installing-ibm-z-k
 ==== {op-system} now includes lsof
 {product-title} {product-version} now includes the `lsof` command in {op-system}.
 
+[id="ocp-4-13-rhcos-new-regions"]
+==== {op-system} images now available in new regions
```

Comment:
@jldohmann While technically accurate, I do not believe the nested bullets are necessary. How about:

"{op-system} images are now available for a few new Alibaba Cloud regions. You can now install {product-title} to the following regions:

* `ap-northeast-2` (Seoul)
* `ap-southeast-7` (Bangkok)

Also - and this is your call, but I am wondering if the content really belongs under Installation and upgrade. From an administrator perspective, I might not think to look under Red Hat Enterprise Linux CoreOS to see if there is anything new with regard to installing an Alibaba cluster. And taking it a step further, do we really need to frame it as "new RHCOS images are available", instead of simply stating that you can now deploy a cluster to the new regions?

Reply:
thanks @mjpytlak yeah, you make a good point. can you PTAL at the new update, and lmk what you think? some of the impetus to mention the RHCOS changes are because of a [support case](https://issues.redhat.com/browse/OSDOCS-4204) that came through about including all RHCOS updates to the RN, but i agree with your opinion about the perspective. i think the new change is a decent compromise